spec/rollbar_spec.rb

```diff
@@ -536,6 +536,82 @@
       Rollbar.error(exception).should == 'disabled'
     end
 
+    context 'using configuration.use_exception_level_filters_default' do
+      before do
+        Rollbar.configure do |config|
+          config.use_exception_level_filters_default = true
+        end
+      end
+
+      context 'without use_exception_level_filters argument' do
+        it 'sends the correct filtered level' do
+          Rollbar.configure do |config|
+            config.exception_level_filters = { 'NameError' => 'warning' }
+          end
+
+          Rollbar.error(exception)
+
+          expect(Rollbar.last_report[:level]).to be_eql('warning')
+        end
+
```

Comment:
This is missing a test when `config.use_exception_level_filters_default = true` has been set, and when `:use_exception_level_filters => false` is passed in with the `Rollbar.error` call.